features/messages/src/main/kotlin/io/element/android/features/messages/MessagesView.kt

```diff
@@ -21,18 +21,7 @@
 
 package io.element.android.features.messages
 
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.Spacer
-import androidx.compose.foundation.layout.WindowInsets
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.imePadding
-import androidx.compose.foundation.layout.navigationBarsPadding
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.statusBars
-import androidx.compose.foundation.layout.width
-import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.*
```

Comment:
ah that's Android Studio decision (probably after I reset some of the .idea files). But maybe we should remove this rule and allow import wildcard. WDYT @jmartinesp and @ganfra ?

Reply:
I don't have a strong opinion for/against wildcard imports. AFAIK the main 2 issues about them are:

1. Cluttering the auto-completion in the IDE.
2. Possibly exporting those symbols if you're working in libraries.

Reason 1 doesn't seem like a problem, but 2 might be an issue if the project grows *a lot* in scale? I'm not sure if it would be noticeable enough, though.

Also, I might be overlooking some other important drawback of using them that I can't remember.